test/specs/utils/stylesheet.js

```diff
@@ -75,4 +78,20 @@ describe('Utils/Stylesheet', function () {
             });
         });
     });
+
+    describe('getRelativeSpriteDir on Windows', function () {
+        var cases = [
+            [ 'test/img/sprite.png', 'test/css/app.css', '../img/sprite.png' ],
```

Comment:
Sorry for the late response, but shouldn't they be Windows paths then? `'test\\img\\sprite.png'`